app/controllers/signup.js

```diff
@@ -1,194 +1,219 @@
-      label: 'Choose a username',
-      type: 'text',
-      errorMessage: 'Username is required',
+      id: "username",
+      label: "Choose a username",
+      type: "text",
+      errorMessage: "Username is required",
       required: true,
-      showError: false
+      showError: false,
     },
     {
-      id: 'email',
-      label: 'Email',
-      type: 'email',
-      errorMessage: 'Valid Email is required',
+      id: "email",
+      label: "Email",
+      type: "email",
+      errorMessage: "Valid Email is required",
       required: true,
       showError: false,
-      validator: this.emailValidator
+      validator: this.emailValidator,
     },
     {
-      id: 'phoneNumber',
-      label: 'Phone Number',
-      type: 'string',
-      value: '+91-',
-      errorMessage: 'Enter a valid phone number',
+      id: "phone_number",
+      label: "Phone Number",
+      type: "string",
+      value: "+91-",
+      errorMessage: "Enter a valid phone number",
       required: false,
       showError: false,
-      validator: this.phoneNumberValidator
+      validator: this.phone_numberValidator,
     },
     {
-      id: 'experience',
-      label: 'Experience',
-      type: 'number',
-      errorMessage: 'Number of experience is required',
+      id: "yoe",
+      label: "yoe",
+      type: "number",
+      errorMessage: "Number of yoe is required",
       required: true,
-      showError: false
+      showError: false,
     },
     {
-      id: 'companyName',
-      label: 'Company Name ',
-      type: 'text',
-      errorMessage: '',
+      id: "company_name",
+      label: "Company Name ",
+      type: "text",
+      errorMessage: "",
       required: false,
-      showError: false
+      showError: false,
     },
     {
-      id: 'designation',
-      label: 'Designation ',
-      type: 'text',
-      errorMessage: '',
+      id: "designation",
+      label: "Designation ",
+      type: "text",
+      errorMessage: "",
       required: false,
-      showError: false
+      showError: false,
     },
     {
-      id: 'github',
-      label: 'Github ',
-      type: 'text',
-      errorMessage: 'GitHub username is required',
+      id: "github_id",
+      label: "Github_id ",
+      type: "text",
+      errorMessage: "GitHub_id username is required",
       required: true,
-      showError: false
+      showError: false,
     },
     {
-      id: 'linkedIn',
-      label: 'LinkedIn ',
-      type: 'text',
-      errorMessage: 'LinkedIn username is required',
+      id: "linkedin_id",
+      label: "linkedin_id ",
+      type: "text",
+      errorMessage: "linkedin_id username is required",
       required: true,
-      showError: false
+      showError: false,
     },
     {
-      id: 'twitter',
-      label: 'Twitter ',
-      type: 'text',
-      errorMessage: 'Twitter handle is required',
+      id: "twitter_id",
+      label: "twitter_id ",
+      type: "text",
+      errorMessage: "twitter_id handle is required",
```

Comment:
Better if we don't show the error message with snake_case in it 😅 
Also once we have mentioned `id` we not need say `id handle required`

Reply:
Changed!